integration_tests/pages/homePage.ts

```diff
@@ -2,7 +2,7 @@ import Page, { PageElement } from './page'
 
 export default class HomePage extends Page {
   constructor() {
-    super('Welcome people into prison', { hasBackLink: false })
+    super('Welcome people into prison', { hasBackLink: false }, { hasFeedbackBanner: true })
```

Comment:
If we're importing `Page` is `hasFeedbackBanner: true` not the default? Not sure this is needed here but could be wrong.